app/renderer/components/add_task.js

```diff
@@ -13,18 +13,23 @@ module.exports = {
         };
     },
     template: `
-    <li class="run-card">
-        <div class="collapsible-header row center-align add-task-header">
-            <strong class="unselectable-text">
-                <span class="small material-icons">add</span>
-                Add New Task
-            </strong>
-        </div>
-        <div class="collapsible-body container task-card-body">
-            <task-input v-on:save="addTask"></task-input>
-        </div>
-    </li>    
+    <ul class="run-card collapsible no-margin" data-collapsible="accordion" >
```

Comment:
This "addtask" was intended to be part of the same accordion, so when opening the AddTask button, it will close the others.

I would prefer to maintain AddTasks as li, instead of a whole new `ul`. I'm not completely sure why you are using a new ul, but if it is needed, you have to close every opened task in that suite

Reply:
When the add button was in the tasks ul tag and it was no task in this list, I could not drag any tasks to it. You can see this if checkout to ddf686a1b4c189bc8c0451c3201684ad25d15d5e. I think around and fix it that way.

Reply:
Any idea why that happens when using the same ul?